test/server.cpp

```diff
@@ -442,13 +436,10 @@ std::string TestContentIn404HtmlResponse::expectedResponse() const
     R"FRAG(</title>
 )FRAG",
 
-    R"FRAG(
-  <link type="root" href="/ROOT"><link type="text/css" href="/ROOT/skin/jquery-ui/jquery-ui.min.css?cacheid=e1de77b3" rel="Stylesheet" />
-<link type="text/css" href="/ROOT/skin/jquery-ui/jquery-ui.theme.min.css?cacheid=2a5841f9" rel="Stylesheet" />
-<link type="text/css" href="/ROOT/skin/taskbar.css?cacheid=49365e9c" rel="Stylesheet" />
-<script type="text/javascript" src="/ROOT/skin/jquery-ui/external/jquery/jquery.js?cacheid=1d85f0f3" defer></script>
-<script type="text/javascript" src="/ROOT/skin/jquery-ui/jquery-ui.min.js?cacheid=d927c2ff" defer></script>
-<script type="text/javascript" src="/ROOT/skin/taskbar.js?cacheid=5982280c" defer></script>
+    R"FRAG(  <link type="root" href="/ROOT"><link type="text/css" href="/ROOT/skin/taskbar.css" rel="Stylesheet" />
```

Comment:
This remove a `\n` as the content starts just after the `R"FRAG(`. This is maybe why the tests are failing.